### DISCUSSION TITLE
Ukrainian doc.noun_chunks is not supported by default?

### DISCUSSION THREAD

**Discussion by notiqq** [explosion/spaCy#7699] [Help: Coding & Implementations] [answered]
Hi,

I was trying to start to use spaCy for dependency parsing for Ukrainian text. As per documentation, it looks like spaCy supports it.  However, as per my tests it requires to install additional statistical model, which is not available by default?

![image](https://user-images.githubusercontent.com/61666187/114083204-5ce6d180-98b7-11eb-9eec-d3c3416591e4.png)

Could you please provide any information on how to fix this?

Thank you!

**Comment by svlandeg** [comment]
Hi! As a maintenance note - this is a good question for the discussion forum, so I'll move it there. The original issue will be closed, but will link through to the new, open discussion thread.

**Comment by polm** [comment, marked as answer]
As described in [Language Support](https://spacy.io/usage/models#languages) in the docs, we have alpha support for Ukranian but not models yet. In order to train a model we would need training data, such as a Universal Dependencies corpus, and possibly some setup code to handle anything unique about the language. 

You can read more about adding languages [here](https://github.com/explosion/spaCy/discussions/3056).
